redis/src/main/java/org/apache/zeppelin/redis/RedisInterpreter.java

```diff
@@ -0,0 +1,98 @@
+package org.apache.zeppelin.redis;
+
+import org.apache.zeppelin.interpreter.Interpreter;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.ERROR;
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.SUCCESS;
+
+public class RedisInterpreter extends Interpreter {
+
+    private Logger LOG = LoggerFactory.getLogger(RedisInterpreter.class);
+
+    private Jedis client;
+
+    private enum RedisCommand {
```

Comment:
yes, i think its better to use this `enum` will be enough.

i was thinking about doing something like:

```
private HashSet<String> redisCommands;

public static HashSet<String> getRedisCommands() {
  HashSet<String> hashSetOfRedisCommands = new HashSet<String>();

  for (Command c : Command.values()) {
      hashSetOfRedisCommands.add(c.name());
  }

  return hashSetOfRedisCommands;
}

// then use this like 
redisCommands.contains(tokens[1]);

```

what do you think?